src/Symfony/Component/Notifier/Bridge/GoogleChat/Tests/GoogleChatTransportFactoryTest.php

```diff
@@ -33,8 +33,8 @@ public function createProvider(): iterable
         ];
 
         yield [
-            'googlechat://chat.googleapis.com/AAAAA_YYYYY?threadKey=abcdefg',
-            'googlechat://abcde-fghij:kl_mnopqrstwxyz%3D@chat.googleapis.com/AAAAA_YYYYY?threadKey=abcdefg',
+            'googlechat://chat.googleapis.com/AAAAA_YYYYY?thread_key=abcdefg',
```

Comment:
I believe, it should be `threadKey` here

Reply:
Nope, this is our DSN format where we want to use snake_case, like all other bridges.

Reply:
indeed, I missed the `googlechat` scheme